### ISSUE TITLE
Clarifying Devfreq Paths

### ISSUE THREAD

**Issue by chphch** [mrsnu/band#293]
We need to check the devfreq paths in resource_monitor.
As far as I know, cdsp-cdsp-l3-lat is devfreq class for memory bandwidth between DSP and L3 cache, which is similar to `cpu4-cpu-l3-lat`.
I'm not so sure, but we need to check.

[band/resource_monitor.cc](https://github.com/mrsnu/band/blob/4eba9847dc5c0a3958015eefebef23927b9940bc/band/resource_monitor.cc#L172C8-L172C8)

**Comment by widiba03304** [comment]
Yes, you are right. AFAIK, DSP's, and NPU's frequency is not adjustable. Need to fix!
